src/store/setToolMode.js

```diff
@@ -23,14 +23,7 @@ const setToolActiveForElement = function (
   const tool = getToolForElement(element, toolName);
 
   if (tool) {
-    _setActiveToolWithMatchingMouseButtonMaskToPassive(tool, element, options);
-    _setActiveToolWithIsTouchActiveToFalse(
-      tool,
-      element,
-      options,
-      isTouchActive
-    );
-
+    _resolveInputConflicts(element, tool, options, isTouchActive);
     // TODO: Find active tool w/ active two finger?
```

Comment:
Not too hot on the touch stuff, but this could all be baked into _resolveInputConflicts also I think? Also, if the method gets too large we could consider putting it in a separate file.